oem.json

```diff
@@ -2870,8 +2870,9 @@
     },
     {
         "id": "com.miui.phrase",
-        "description": "Frequent Phrases\nNot sure how exactly it can be used but it is supposed to predict phrases you'll want to write.\nI don't know why it isn't handled in the keyboard app. This seems to be something else.\nIn any case it has access to internet, is linked to MiCloud and contains a weird CloudTelephonyManager java class in his code.\n",
-        "removal": "delete"
+        "description": "MIUI context menu tool for pre-made text responses while chatting.",
```

Comment:
You've left out some important information: (unless they're wrong, of course)

> it has access to internet, is linked to MiCloud and contains a weird CloudTelephonyManager java class in his code.

Reply:
Yes. I should have left this information. 

I focused on the correct description and on a serious error in the system context menu.

Reply:
Add `label` Frequent Phrases